### PR TITLE
15st1 [2848][ADD] partner_attribute: add tests

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[2848](https://www.quartile.co/web#id=2848&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: partner_attribute/tests/test_partner_birth_month.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        self.partner_admin.write({"birthday": "1991-09-05"})
+
+    def test_compute_birth_month(self):
+        self.partner_admin._compute_birth_month()
+        birth_month = int(
+                    datetime.strptime(
+                        str(self.partner_admin.birthday), DEFAULT_SERVER_DATE_FORMAT
+                    ).strftime("%m")
+                )
+        self.assertEqual(self.partner_admin.birth_month, birth_month)
```
Comment (yostashiro) [inline partner_attribute/tests/test_partner_birth_month.py]: This test is pointless if you copy and paste the content of the compute method.  It's not proving anything.

**Review by yostashiro — CHANGES_REQUESTED** [review]
Please also rename the file to test_partner_attribute so that test cases can be added to the same file.

File: partner_attribute/tests/test_partner_birth_month.py (hunk 1 above)
Comment (AungKoKoLin1997) [reply partner_attribute/tests/test_partner_birth_month.py]: When I learn how to write tests from oca repos (https://github.com/OCA/partner-contact/blob/a444fe58d32b7db42fdbb8170be030022ff7fcb4/partner_contact_birthdate/tests/test_res_partner.py#L16-L21), I found some module write tests like this. So, I thought using same content is not the problem.

**Review by yostashiro — CHANGES_REQUESTED** [review]
Please fix the pre-commit error.

**Comment by yostashiro** [comment]
@AungKoKoLin1997 Please rebase your branch to run the tests.

**Review by yostashiro — APPROVED** [review]

**Pull request closed without merging**